app/views/Settings.js

```diff
@@ -123,6 +123,11 @@ export const SettingsScreen = ({ navigation }) => {
             label={t('share.title')}
             description={t('share.subtitle')}
             onPress={() => navigation.navigate('ExportScreen')}
+          />
+          <Item
```

Comment:
This should be wrapped in a Feature Flag for the time being (please see my main comment).

```js
<Feature name='self_assessment'>
  <Item
    label={t('assessment.settings_title')}
    description={t('assessment.subtitle')}
    onPress={() => navigation.navigate('AssessmentScreen')}
     last
   />
</Feature> 
```

You will also need to add an entry to `.env`:

```
flag_self_assessment=false
```